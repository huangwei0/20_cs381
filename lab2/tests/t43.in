debug off
stopwords off
length 10
load tracks.txt
preprocess
sequence pearl
sequence problems