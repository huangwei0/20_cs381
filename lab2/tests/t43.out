pearl of the world is a little girl in my heart
problems 2 in the world is a little girl from me
